realm/realm-library/src/objectServer/java/io/realm/SyncConfiguration.java

```diff
@@ -25,6 +25,7 @@
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 import java.util.Arrays;
+import java.util.Collections;
```

Comment:
It looks like it is added accidently. It is never used in `SyncConfiguration.java`.